packages/daimo-api/src/shovel/watcher.ts

```diff
@@ -0,0 +1,23 @@
+import { Pool } from "pg";
+
+type callback = (pg: Pool, from: bigint, to: bigint) => void | Promise<void>;
+
+export class Watcher {
```

Comment:
i'd add a comment describing how Watcher works

what happens when a new block appears, etc

Reply:
100%. This is very much a sketch. Once things start to solidify I will add some docs.